functional/README.md

```diff
@@ -1,3 +1,63 @@
+## configure environment
+
+### Debian/Ubuntu
+
+#### Install Vagrant
+
+```sh
+sudo apt-get install -y git nfs-kernel-server
+wget https://releases.hashicorp.com/vagrant/1.8.1/vagrant_1.8.1_x86_64.deb
+sudo dpkg -i vagrant_1.8.1_x86_64.deb
+```
+
+#### Install VirtualBox
+
+```sh
+echo "deb http://download.virtualbox.org/virtualbox/debian $(lsb_release -sc) contrib" | sudo tee /etc/apt/sources.list.d/virtualbox.list
+wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | sudo apt-key add -
+sudo apt-get update
+sudo apt-get install -y build-essential dkms
+sudo apt-get install -y VirtualBox-5.0
+#Previous VirtualBox (if you have problems with nested virtualization, more info here: https://www.virtualbox.org/ticket/14965)
+#sudo apt-get install -y VirtualBox-4.3
+```
+
+### Centos/Fedora
+
+**NOTE**: NFS doesn't work on CentOS 6.x
```

Comment:
Don't want to waste my time trying to debug why it doesn't work in old CentOS 6. So yes.